scripts/generate-app-config/generateContentConfig.js

```diff
@@ -0,0 +1,225 @@
+const kebabCase = require('lodash.kebabcase');
+const startCase = require('lodash.startcase');
+const ImportAggregator = require('./generation-objects/ImportAggregator');
+
+// "require" items to be added to the generated config.
+const ContentWrapper = 'terra-dev-site/lib/wrappers/_ContentWrapper';
+const MarkdownWrapper = 'terra-dev-site/lib/wrappers/_MarkdownWrapper';
+const TerraScreenshotWrapper = 'terra-dev-site/lib/wrappers/_ScreenshotWrapper';
+
+const routeImporter = new ImportAggregator();
+/**
+* Setup a menuItem object.
+*/
+const menuItem = (text, itemPath, hasSubMenu, childItems) => ({
+  text,
+  path: itemPath,
+  childItems,
+  ...(hasSubMenu) && { hasSubMenu: true },
+});
+
+/**
+ * Builds out a route item. Adds the props object conditionally.
```

Comment:
Might be helpful to rename text to linkName or something. Assuming I'm understanding the intent of text here

Reply:
renamed